documentation/proposals/Proposal - Multi-Backend Input.md

```diff
@@ -834,6 +831,103 @@ public interface IJoystickInputHandler : IInputHandler
 
 `HandleHatMove` must be called when any value of `JoystickState.Hats` changes.
 
+# Input Maps
+
+## Presentation
+Input maps are designed to be easyer to the user to handle actions with more than one input.
+At first view it can looks useless and a bad practice, but this is complemtelly useful for cross-plataform programs, being games an example.
```

Comment:
This is a useless comment and contains unfounded claims you have not backed up.
- Why does it look useless?
- Why does it look like bad practice?
- Why specifically is it useful, rather than just saying it's useful?
- Why is it useful for games specifically?
- Why is it useful for cross-platform programs specifically?

Reply:
idk, ask for beyley, she said it or something of the kind lol

Reply:
What i had said "looks useless and bad practice" was the combination of multiple events into a single callback, i dont see a reason to combine keyboard and mouse events into a single callback, when you'd have to `if(ev is KeyboardEvent kbEvent)`, im not against input *maps* being in Silk.NET (_as an extension_, **not** core)

what i dont like is `public event Action<dynamic>? InputEvent;`

Reply:
Oh, reality? Sry so, my fault. I will fix it

Reply:
> So to clarify, you’re making a proposal you don’t personally understand the usefulness in?

I just don't see anything useless on it, i just misunderstood Beyley's comment. But anyway you have to say, it really looks a little useless at first look lol